packages/react/src/components/AccountSettings/ChangePassword/ChangePassword.tsx

```diff
@@ -177,7 +177,11 @@ function ChangePassword({
   }
 
   return (
-    <View as="form" className="amplify-changepassword" onSubmit={handleSubmit}>
+    <View
+      as="form"
+      className="amplify-accountsettings-changepassword"
```

Comment:
Would be nicer if instead of hardcoding, these class names were constants, could be used in unit tests as well, to assert they are applied as expected ;)
Ditto for the other classnames as well.

Reply:
Good point. I've added `ComponentClassName` constant. I didn't put it in ui package yet. 

I think we should refactor / consolidate connected components classnames to the ui package (especially authenticator ones are all the place), but let's keep that to a different PR.